src/Tribe/PUE/Checker.php

```diff
@@ -123,14 +164,15 @@ public function hooks() {
 			// Key validation
 			add_action( 'wp_ajax_pue-validate-key_' . $this->get_slug(), array( $this, 'ajax_validate_key' ) );
 
-			// Dashboard message "dismiss upgrade" link
-			add_action( 'wp_ajax_' . $this->dismiss_upgrade, array( $this, 'dashboard_dismiss_upgrade' ) );
-
 			add_filter( 'tribe-pue-install-keys', array( $this, 'return_install_key' ) );
 
 			add_action( 'admin_enqueue_scripts', array( $this, 'maybe_display_json_error_on_plugins_page' ), 1 );
 
-			tribe_notice( 'pue-validation', array( $this, 'display_license_error_message' ), 'dismiss=1&type=warning' );
+			tribe_notice( 'license-invalid-validation', __CLASS__ . '::is_api_invalid_warning', 'dismiss=1&type=warning' );
```

Comment:
Could we use the `array` structure instead of concat.

```php
array( self, 'is_api_invalid_warning' )
// or
array( __CLASS__, 'is_api_invalid_warning' )
```